vector/src/main/res/drawable/ic_settings_root_advanced.xml

```diff
@@ -1,5 +1,5 @@
 <vector android:autoMirrored="true" android:height="24dp"
     android:viewportHeight="24" android:viewportWidth="24"
     android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
-    <path android:fillColor="?riotx_header_panel_text_secondary" android:fillType="evenOdd" android:pathData="M22,12C22,17.5228 17.5228,22 12,22C6.4771,22 2,17.5228 2,12C2,6.4771 6.4771,2 12,2C17.5228,2 22,6.4771 22,12ZM8.5,12C8.5,12.8284 7.8284,13.5 7,13.5C6.1716,13.5 5.5,12.8284 5.5,12C5.5,11.1716 6.1716,10.5 7,10.5C7.8284,10.5 8.5,11.1716 8.5,12ZM12.25,13.5C13.0784,13.5 13.75,12.8284 13.75,12C13.75,11.1716 13.0784,10.5 12.25,10.5C11.4216,10.5 10.75,11.1716 10.75,12C10.75,12.8284 11.4216,13.5 12.25,13.5ZM19,12C19,12.8284 18.3284,13.5 17.5,13.5C16.6716,13.5 16,12.8284 16,12C16,11.1716 16.6716,10.5 17.5,10.5C18.3284,10.5 19,11.1716 19,12Z"/>
+    <path android:fillColor="@color/riotx_header_panel_text_secondary_light" android:fillType="evenOdd" android:pathData="M22,12C22,17.5228 17.5228,22 12,22C6.4771,22 2,17.5228 2,12C2,6.4771 6.4771,2 12,2C17.5228,2 22,6.4771 22,12ZM8.5,12C8.5,12.8284 7.8284,13.5 7,13.5C6.1716,13.5 5.5,12.8284 5.5,12C5.5,11.1716 6.1716,10.5 7,10.5C7.8284,10.5 8.5,11.1716 8.5,12ZM12.25,13.5C13.0784,13.5 13.75,12.8284 13.75,12C13.75,11.1716 13.0784,10.5 12.25,10.5C11.4216,10.5 10.75,11.1716 10.75,12C10.75,12.8284 11.4216,13.5 12.25,13.5ZM19,12C19,12.8284 18.3284,13.5 17.5,13.5C16.6716,13.5 16,12.8284 16,12C16,11.1716 16.6716,10.5 17.5,10.5C18.3284,10.5 19,11.1716 19,12Z"/>
```

Comment:
is it ok to use light here? is there any tint apply after?

Reply:
yes, in the same PR :)